### DISCUSSION TITLE
Constant ur and ul values

### DISCUSSION THREAD

**Discussion by MarcColl10** [tudelft/AE4353-Y24#17] [Ex_1] [answered]
After training the model, my model only predicts with a constant ur and ul values, which gives me the following result, how can I solve this?
![AI ex1](https://github.com/user-attachments/assets/0cee94d9-ab5c-450f-a6f9-432a52090e3f)

**Comment by yilun-wu** [comment]
I would debug why your model gives a constant output. Try to use a debugger to see how the constant output is produced from your model's forward pass.

**Reply by MarcColl10** [reply, marked as answer]
Nevermind, I have the learning rate 3 orders of magnitude bigger than it should. After setting lr=0.001 it worked.

**Reply by Huizerd** [reply]
good job! usually, learning rates are in the range of 1e-2 to 1e-5, so it most of the times works to start there. if it doesn't seem to learn, go 10x higher and 10x lower, and observe and think about what happens :)
